cli/flox/src/commands/build.rs

```diff
@@ -165,3 +221,41 @@ fn available_packages(lockfile: &Lockfile, packages: Vec<String>) -> Result<Vec<
 
     Ok(packages_to_build)
 }
+
+#[cfg(test)]
+mod test {
+    use flox_rust_sdk::flox::test_helpers::flox_instance;
+    use flox_rust_sdk::models::environment::path_environment::test_helpers::new_path_environment_in;
+    use flox_rust_sdk::providers::buildenv::NIX_BIN;
+    use tempfile::tempdir_in;
+
+    use super::*;
+
+    #[test]
+    /// Test that check_and_display_symlink shortens the symlink when in the
+    /// current directory,
+    fn symlink_gets_shortened_when_in_current_dir() {
+        let (flox, _temp_dir) = flox_instance();
+        let dot_flox_parent_path = tempdir_in(&flox.temp_dir)
+            .unwrap()
+            .into_path()
+            .canonicalize()
+            .unwrap();
+        let environment = new_path_environment_in(&flox, "version 1", &dot_flox_parent_path);
```

Comment:
This could be done with the existing helper:
```suggestion
        let environment = new_path_environment(&flox, "version 1");
        let dot_flox_parent_path = environment.parent_path().unwrap();
```

Reply:
My thought process was that we want to confirm that if you create an env in a directory and then build in that directory, the result symlink gets shortened. That's ever so slightly more focused on behavior and less of a circular check that the directory gets shortened when the environment thinks cwd matches `environment.parent_path()`. Happy to approve followup if you think that's too pointless of a distinction